src/bsp/src/stm32/src/BSP.cpp

```diff
@@ -10,7 +11,4 @@ void BSP::initChip(void* args) {
     Hal_init();
 }
 
-uint16_t BSP::getUUId() const {
-    // TODO: Change do the ID is obtained from persistent memory
-    return 1;
-}
\ No newline at end of file
+uint16_t BSP::getUUId() const { return SettingsContainer::GetUUID(); }
```

Comment:
Just to be sure, the plan here is to eventually use the UUID from the EPROM and if there is none, we will store and use the UUID from the settings container?

Reply:
Exactly